### DISCUSSION TITLE
Leasson 6 Unable to deploy to mainnet-fork-dev

### DISCUSSION THREAD

**Discussion by shlok-codes** [smartcontractkit/full-blockchain-solidity-course-py#638] [Q&A] [answered]
Hello,

I am pretty sure i created the miannet-fork-dev correctly, but can't seem to launch deploy.py there?
I get the following error:
![image](https://user-images.githubusercontent.com/30735595/147423754-91b2f3b6-cae9-405c-9ac6-1435ff971708.png)

Here is proof that I created the fork correctly after I type in brownie networks list:

![image](https://user-images.githubusercontent.com/30735595/147423866-51ad8ea7-adec-4faa-9ffe-ce77ef667f95.png)

Thank you for your help.

**Comment by cromewar** [comment, marked as answer]
Hello @shlok-codes is mainnet-fork-dev on your helpful scripts file? as we use `network.show_active()`you should also add it to your helpful scripts, other wise it won't work.

**Reply by shlok-codes** [reply]
Hi @cromewar. Yes sir. It is there as well:
![image](https://user-images.githubusercontent.com/30735595/147424263-a974b0e4-caa3-4350-93e1-c4e99287d749.png)

**Reply by cromewar** [reply]
It seems you are on Mac, could you delete that network and add it again? and also share the command you use? this is quite strange.
Additionally do you have different python interpreters? make sure you are using the same one on the terminal and on vscode.

**Reply by shlok-codes** [reply]
Hi, thanks for your continuous support.
I am not sure how to delete a network. Could you tell me the command please?

For adding the mainnnet fork, I used the same command as Patrick with my own alchemy http of course. 

The python interpreters are the same.

**Reply by cromewar** [reply]
Somethings you could have troubles even with a simple comma or a space, to delete the network use:

`brownie networks --delete mainnet-fork-dev`

Then restart the terminal, and try adding it again, but double check everything.

**Reply by shlok-codes** [reply]
That was it my friend. When I went to delete the network I couldn't. And I notices that I had spelled the mainnet with three 'n's. EG mainnnet.

Anyway it is fixed and working. Thank you again!

**Reply by cromewar** [reply]
I'm glad is solved now.
